<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineAddSelectionWidget</class>
 <widget class="QWidget" name="LineAddSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Add points mode: Click in the video to add points to the end of the selected line.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="smoothingGroupBox">
     <property name="title">
      <string>Smoothing Method</string>
     </property>
     <layout class="QVBoxLayout" name="smoothingLayout">
      <item>
       <widget class="QRadioButton" name="simpleSmoothRadio">
        <property name="text">
         <string>Simple Smoothing</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="polyFitRadio">
        <property name="text">
         <string>Polynomial Fit</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="polyOrderLayout">
        <item>
         <widget class="QLabel" name="polyOrderLabel">
          <property name="text">
           <string>Polynomial Order:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="polyOrderSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="edgeSnappingCheckbox">
     <property name="toolTip">
      <string>Automatically snap points to nearby edges in the image</string>
     </property>
     <property name="text">
      <string>Snap to edges</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="edgeGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Edge Detection Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="edgeLayout">
      <item>
       <layout class="QHBoxLayout" name="thresholdLayout">
        <item>
         <widget class="QLabel" name="thresholdLabel">
          <property name="text">
           <string>Threshold:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="thresholdSlider">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="thresholdValueLabel">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="radiusLayout">
        <item>
         <widget class="QLabel" name="radiusLabel">
          <property name="text">
           <string>Search Radius (px):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="radiusSpinBox">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
